brownie/project/compiler/__init__.py

```diff
@@ -1,19 +1,19 @@
 #!/usr/bin/python3
 
-import json
 from copy import deepcopy
 from hashlib import sha1
+import json
 from pathlib import Path
 from typing import Dict, Optional
 
-import solcast
 from eth_utils import remove_0x_prefix
 from semantic_version import Version
+import solcast
 
 from brownie._config import _get_data_folder
 from brownie.exceptions import UnsupportedLanguage
 from brownie.project import sources
-from brownie.project.compiler.solidity import (  # NOQA: F401
+from brownie.project.compiler.solidity import (
     find_best_solc_version,
     find_solc_versions,
     install_solc,
```

Comment:
still failing the lint CI..  i think everything that's changed in this section needs to be undone.

Reply:
Yeah, my mistake, returned it back

Reply:
still need to change some of this back to how it was.  you might consider running `tox -e lint` locally to check everything.